alembic/versions/2023_12_11_79454ff320bf_add_eligible_for_skip_qc.py

```diff
@@ -0,0 +1,27 @@
+"""add-eligible-for-skip-qc
+
+Revision ID: 79454ff320bf
+Revises: b105b426af99
+Create Date: 2023-12-11 15:02:40.593762
+
+"""
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "79454ff320bf"
+down_revision = "b105b426af99"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "application",
+        sa.Column("is_eligible_for_skip_qc", sa.Boolean(), nullable=False),
```

Comment:
Might be necessary to provide a default value to the column here as well?

Reply:
Apparently not

Reply:
but maybe a good idea to set it to False as default, not sure.

Reply:
A bit unsure what it would add, given that the script has already been tested and added the column with False as default.